apps/server/src/modules/files-storage/helper/file.ts

```diff
@@ -0,0 +1,19 @@
+import { AxiosResponse } from 'axios';
+import { Request } from 'express';
+import { Readable } from 'stream';
+import { IFile } from '../interface';
+
+export function createFile(
+	filename: string,
+	transaction: Request | AxiosResponse,
+	buffer: Buffer | Blob | ReadableStream | Readable
+): IFile {
+	const file: IFile = {
+		name: decodeURI(filename),
+		buffer,
+		size: Number(transaction.headers['content-length']),
+		mimeType: transaction.headers['content-type'] || '',
```

Comment:
@SevenWaysDP  @CeEv createFile kann nun für express requests und axios response verwendet werden. Das einzige Problem ist das transaction.headers['content-type'] undefined werden kann. Wie könnte man damit umgehen? Für transaction.headers['content-length'] gilt wahrscheinlich das gleiche.

Reply:
Wurde in FileBuilder geändert